rcongui/src/components/SettingsView/vips.js

```diff
@@ -182,9 +173,7 @@ const VipEditableList = ({
 }) => {
   const [name, setName] = React.useState("");
   const [steamID64, setSteamID64] = React.useState("");
-  const [expirationTimestamp, setExpirationTimestamp] = React.useState(
-    moment().add(30, "days").format()
-  );
+  const [VIPPlayer, setVIPPlayer] = React.useState(false);
```

Comment:
I would have set an empty object or undefined as the default here. Using false as the default made me though this variable was meant to be a boolean flag

Reply:
I thought about this when I did the changes as well. However, I then took what was done in the Playerhistory for now. It seems I totally forgot revisiting this part and re-considering is a boolean is really a good idea.